recipes/reaper/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "REAPER" %}
+{% set version = "e22f2dbdad3af8b725d3fc25e6c96b23ae9aff70" %}
```

Comment:
This won't work well in conda, since these versions can't be compared to one another.

I guess there aren't actually any version numbers for this. That's...unfortunate. Maybe a date-based version is the best we can do....

Reply:
So maybe a version 2018.01?

Reply:
Maybe `0.0.2018.01.13` or something, so that if in the future there's e.g. a `0.1` release it'll sort properly?